promptulate/agents/tool_agent/agent.py

```diff
@@ -160,32 +161,37 @@ def _should_continue(self, current_iteration: int, current_time_elapsed) -> bool
             return False
         return True
 
-    def _find_action(self, answer: str) -> (str, str):
+    def _parse_llm_response(self, answer: str) -> (str, str):
         """Parse next instruction of LLM output.
 
         Args:
             answer(str): output of LLM
 
         Returns:
-            Return a tuple, (action, action input)
+            Return a tuple, (thought,action, action input)
```

Comment:
(thought, action, action input)  Pay attention to spaces

Reply:
ok